mslib/msidp/idp_user.py

```diff
@@ -133,3 +58,7 @@
         "schacUserPresenceID": "skype:pepe.perez",
```

Comment:
when that not be needed to be named "skype"  I would prefer "sky"

The background of  that request is, that we recently got requests by scanning tools finding strings of proprietary software on our systems. This tools create noise because of possible license issues.